### ISSUE TITLE
sae_manifold_fit: continuation pre-warm always falls back to cold seed (β length 0 != decoder dim) → very slow REML at tiny scale

### ISSUE THREAD

**Issue by SauersML** [SauersML/gam#579]
> _Migrated from SauersML/Manifold-SAE#1 — this is a gamfit `sae_manifold_fit` / arrow-Schur REML issue, not Manifold-SAE-side. `ManifoldSAE.fit()` delegates to `gamfit.sae_manifold_fit` → `sae_manifold_fit_minimal`._

Observed against **gamfit 0.1.141**.

## Summary

`ManifoldSAE.fit()` (the closed-form REML SAE-manifold solve, invoked by
`extract_feature_curves` at eval time and by any `fit_every>0` training) is
**very slow even at tiny scale** — minutes for a single solve on a toy problem
(D=32, 6 atoms, ≤1500 rows). It dominates experiment wall-clock and makes
interactive synthetic-recovery runs impractical on CPU.

## Repro

`experiments/orthogonal_manifolds.py` at tiny scale:
`d_ambient=32, 4 manifolds + 2 slack atoms, intrinsic_rank=2, n_basis=10,
probe_fit_rows=1500`. Training (200 backprop steps) runs in seconds; the single
eval-time `sae.fit()` then runs for **minutes**.

A standalone `sae.fit(x)` over 6000 rows (D=64, 10 atoms) did not finish within
2+ minutes (heartbeats at 1m, 2m, still going).

## What the gamfit log shows

The continuation pre-warm fails on every seed and falls back to a cold solve,
and several seeds are rejected for ill-conditioning:

```
[OUTER] SAE manifold: continuation pre-warm for seed N did not complete
  (seed_inner_state: β length 0 != decoder dim 6144
   (continuation stuck at oversmooth offset 2.772589e1, final rho dim=14));
  falling back to a cold seed eval
[OUTER] SAE manifold: rejecting seed 4 before solver start:
  REML smoothing optimization failed to converge: ... arrow-Schur:
  per-row H_tt^(0) Cholesky ... ill-conditioned (kappa_estimate=9.36e7);
  Schur reduction would be numerically contaminated
[SAE-AUDIT] decoder atom 'atom_k' weighted design is rank-deficient
  (rank=18/32, 14 weakly-identified column(s), n=1500)
```

So per outer call we: (a) attempt continuation pre-warm, (b) detect
`β length 0 != decoder dim` and fall back to cold, (c) cold-solve a
rank-deficient, ill-conditioned design — repeated across multiple seeds.

## Likely root cause / asks

- The `β length 0 != decoder dim` mismatch makes the continuation **always**
  fail its pre-warm and fall back to a full cold solve every seed — looks like a
  warm-start state isn't being seeded, so we pay full cost N_seeds times.
- Rank-deficient per-atom designs (rank 18–24 / 32) at small n drive the
  ill-conditioning that rejects seeds. With `intrinsic_rank=2` the atom only
  needs a 2-D ambient subspace; the (n_basis × D) decoder block is wildly
  over-parameterized for the data, so the weighted design is structurally
  rank-deficient. Could the solve project to the atom's active subspace first?

### Suggestions
1. Fix the continuation warm-seed so pre-warm actually completes (kill the
   per-seed cold-solve fallback) — likely the biggest win.
2. Allow capping/short-circuiting the seed sweep (e.g. `n_seeds=1`) for small
   problems via `RemlConfig`.
3. Surface a fast path / tolerance knob for toy-scale fits.

## Environment
- gamfit 0.1.141, torch 2.12.0, CPU (CUDA not compiled in), macOS arm64.

Workaround in-repo: `experiments/orthogonal_manifolds.py` now caps the eval
probe solve to `probe_fit_rows` (default 1500) instead of the full dataset.

**Comment by SauersML** [comment]
🔒 claim: claude-code coordinator session — taking this now. (cluster: sae-reml-core)
